services/QuillCMS/app/workers/batch_reindex_responses_worker.rb

```diff
@@ -6,11 +6,11 @@ class BatchReindexResponsesWorker
 
   def perform(start_index, end_index)
     responses = Response.where(id: start_index..end_index)
-    Response.__elasticsearch__.client.bulk(
+    Response.__elasticsearch__.client.bulk({
```

Comment:
The addition of `{` and `}` seems [unnecessary](https://makandracards.com/makandra/496481-changes-to-positional-and-keyword-args-in-ruby-3-0#section-what-changed)

Since the tests stub out these methods, can you verified on staging that these are required?